package.json

```diff
@@ -85,6 +85,7 @@
 		"eslint-plugin-react": "7.7.0",
 		"extract-text-webpack-plugin": "4.0.0-beta.0",
 		"node-sass": "git://github.com/sass/node-sass.git#v4.7.0",
+		"npm-run-all": "4.1.3",
```

Comment:
What is `npm-run-all` providing that isn't already possible with `concurrently` (also a dependency)? Can we go with one or the other and not both?